lib/paper_trail/version_concern.rb

```diff
@@ -218,7 +218,7 @@ def changeset
 
     # Returns who put the item into the state stored in this version.
     def paper_trail_originator
-      @paper_trail_originator ||= previous.whodunnit rescue nil
+      @paper_trail_originator ||= previous && previous.whodunnit
```

Comment:
I'd rather use `try` here. PT still depends on activesupport >= 3, so we can't use `try!`.